### PR TITLE
Use new data table style

### PR DESCRIPTION
**Pull request by bramkragten**
Use the new data table style, moves the content of the filter dialog to the filter pane.
Adds grouping by state and type.
Removes the default filtering by downloaded and removing the filter when starting to search.
Removes the FAB and dialog saying it has been removed.

Filters will be updated in a follow up PR

CleanShot 2024-04-17 at 12 37 36

CleanShot 2024-04-17 at 12 38 17

### CODE REVIEW

File: src/dashboards/hacs-dashboard.ts [hunk 1]
```diff
@@ -85,16 +88,18 @@ const FILTER_SCHEMA = memoize(
...
         selector: {
           select: {
-            options: BASE_FILTER_OPTIONS.map((filter) => ({
-              value: filter,
-              label: localizeFunc(
-                // @ts-ignore
-                `common.${filter}`,
-              ),
-            })),
+            options: ["pending-restart", "pending-upgrade", "installed", "new", "default"].map(
```
Comment (ludeeus) [inline src/dashboards/hacs-dashboard.ts:94]:
```suggestion
            options: Object.keys(STATUS_ORDER).map(
```
To ennsure that they are in sync.

**Review by piitaya — COMMENTED** [review]
Code wise it seems ok for me. I will pull the repository to do functional tests 🙂

File: src/localize/languages/en.json [hunk 2]
```diff
@@ -28,14 +28,17 @@
...
+  },
   "column": {
     "name": "Repository Name",
     "downloads": "Downloads",
     "stars": "Stars",
     "installed_version": "Downloaded version",
     "available_version": "Available version",
     "status": "Status",
-    "category": "Category",
+    "category": "Type",
```
[the remark this answers is not in the document]
Comment (bramkragten) [reply src/localize/languages/en.json:41]: It should probably be changed in the backend too, to align it?

[the remark this answers is not in the document]
Comment (piitaya) [reply src/localize/languages/en.json:41]: I think so

**Review by piitaya — APPROVED** [review]
I tested the changes in my dev instance and I don't see any issue. It works well 🙂

**Review by ludeeus — APPROVED** [review]
Awesome!

**Pull request merged**